### PR TITLE
move from TravisCI to CircleCI

### PR DESCRIPTION
**Pull request by afeld**
* [x] Get NPM modules installed in deployed app when deploying with autopilot
* [x] Limit deployments to `master`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          name: Set up Code Climate test coverage reporter
+          command: ./cc-test-reporter before-build
+
+      - run:
+          name: Run tests
+          command: bundle exec rspec
+
+      - run:
+          name: Submit test report to Code Climate
+          command: ./cc-test-reporter after-build --exit-code $?
```
Comment (adborden) [inline .circleci/config.yml:45]: Did you verify that this works? I wouldn't assume the exit code gets passed along. If not you could join them
```
command: |
  bundle exec rspec
  ./cc-test-reporter after-build --exit-code $?
```

Or maybe it's just magic and all is good :)

**Review by adborden — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (afeld) [reply .circleci/config.yml:45]: Pretty sure test coverage is only acknowledged by Code Climate on the primary branch...cool if we merge, then correct after if it doesn't work?

Comment (adborden) [reply .circleci/config.yml:45]: Of course!

**Comment by adborden** [comment]
FYI, I just migrated the docker image to 18fgsa and created a branch [adb-circleci](https://github.com/18F/compliance-viewer/pull/85) with the change.

**Pull request merged**